web/web.go

```diff
@@ -111,6 +112,8 @@ func NewWebService(o *WebServiceOptions) *WebService {
 		router.Post("/-/quit", ws.quitHandler)
 	}
 
+	router.Get("/federate", instr("federate", o.Federation))
```

Comment:
Ok... well, let's put it under `/federate` for now and maybe revisit before the next release.